package.json

```diff
@@ -29,7 +30,7 @@
   },
   "dependencies": {
     "express": "^4.17.2",
-    "hydrogen-view-sdk": "^0.0.4",
+    "hydrogen-view-sdk": "npm:@mlm/hydrogen-view-sdk@^0.0.12",
```

Comment:
Custom version `hydrogen-view-sdk` which runs the scratch changes from https://github.com/vector-im/hydrogen-web/pull/653 so everything just works out of the box